nix/fetch-bky-as.sh

```diff
@@ -17,13 +18,15 @@ else
     echo "no current version"
 fi
 
-echo -n "Getting the latest commit..."
-latest_commit=$(gh api repos/blocky/delphi/commits --jq '.[0].sha')
-echo "'$latest_commit'"
+if [[ $commit == "latest" ]]; then
+  echo -n "Getting the latest commit..."
+  commit=$(gh api repos/blocky/delphi/commits --jq '.[0].sha')
+  echo "'$commit'"
+fi
```

Comment:
How do you feel about this approach to grabbing the latest sha? We pass in the commit as the string "latest" and then fetch that commit. Otherwise we assume the commit being passed in is a valid git commit.